Amazon.IonDotnet.Tests/Internals/TextReaderTest.cs

```diff
@@ -210,6 +200,15 @@ public void HasAnnotationFalse_SingleField()
             ReaderTestCommon.HasAnnotationFalse_SingleField(reader);
         }
 
+        [TestMethod]
+        public void HasAnnotation_AssertUnknownSymbolException()
+        {
+            byte[] data = DirStructure.OwnTestFileAsBytes("text/unknown_symbols.ion");
```

Comment:
Can you just put the data in a string literal and not have to use the file, like you do in the other tests?

Reply:
Fixed. I've noticed the other text tests have been opting to load text through a Ion file, so I was following the same pattern.